packages/documentation/docs/controls/_avatar_code.md

```diff
@@ -32,7 +32,7 @@ You can also add the avatar to the header, which will turn it into a clickable b
 
 <Playground 
   name="application-header" 
-  examplesByName 
+   
```

Comment:
remove empty line

Reply:
It was a prettier auto formatting will be adapted with file change to `.mdx` file. Resolve other too